client/js/lounge.js

```diff
@@ -729,6 +729,7 @@ $(function() {
 		var self = $(this);
 		viewport.toggleClass(self.attr("class"));
 		e.stopPropagation();
+		chat.find(".chan.active .chat").trigger("msg.sticky");
```

Comment:
https://github.com/thelounge/lounge/blob/ecb60e3c787511d175f4d2f29b41fbadd7ba08ab/client/js/libs/jquery/stickyscroll.js#L13-L17 and line 40 for the event.